controllers/lvmcluster_controller.go

```diff
@@ -95,7 +95,11 @@ func (r *LVMClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 // errors returned by this will be updated in the reconcileSucceeded condition of the LVMCluster
 func (r *LVMClusterReconciler) reconcile(ctx context.Context, instance *lvmv1alpha1.LVMCluster) (ctrl.Result, error) {
-	resourceList := []resourceManager{}
+	// TODO: Resource deletion should be in reverse order of install
+	resourceList := []resourceManager{
```

Comment:
This will need to be split up amongst the 2 PRs (CSIDriver and topolvm-controller)